system/LINK/etc/default/evm

```diff
@@ -28,3 +28,12 @@ export KEY_ROOT=/var/www/miq/vmdb/certs
 [[ -s /opt/rh/postgresql92/enable ]] && source /opt/rh/postgresql92/enable
 [[ -s /opt/rh/nodejs010/enable ]] && source /opt/rh/nodejs010/enable
 [[ -s /etc/default/evm_productization ]] && source /etc/default/evm_productization
+
+if [[ -s /opt/rh/ruby200/enable ]]
+  source /opt/rh/ruby200/enable
+else
+  rubies="/opt/rubies/ruby-2.0.0*/bin"
+  # add latest (alphabetically) ruby version
+  export PATH=$PATH:${rubies[${#rubies[@]} - 1]}
+fi
```

Comment:
No that's wrong too :)

We don't have overlay support for productization, we have "switching" support.

The file you create downstream should be named something else...then there needs to be something that does

psuedocode:
if file.exists?(evm_productization)
  source evm_productization
else
  source evm_this_new_file_we_are_talking_about
fi